commerce-price-list-service/service.xml

```diff
@@ -74,7 +74,7 @@
 
 		<!-- Other fields -->
 
-		<column name="commerceCurrencyId" type="long" />
+		<column name="commerceCurrencyCode" type="String" />
```

Comment:
`commerceCurrencyCode` should be sorted below `parentCommercePriceListId` (and probably below `name` as well), since it's not a foreign key.